### DISCUSSION TITLE
If postgres is used as persistence...  does conductor uses dyno-queues for queuing?

### DISCUSSION THREAD

**Discussion by santhosh34** [Netflix/conductor#3631] [Q&A]
Trying to understand the concept of queueing for mysql and postgres persistence in conductor. 

As per the documentation: https://conductor.netflix.com/devguide/architecture/index.html
conductor  uses [dyno-queues](https://github.com/Netflix/dyno-queues) internally. 

1)  dyno queue will be in action only if  dynamite is used as persistence for conductor?
2) if we use postgres or mysql, whats the default queuing mechanism? does it still uses dyno -queue? 

Referred the docs:
https://netflixtechblog.com/distributed-delay-queues-based-on-dynomite-6b31eca37fbc
https://github.com/Netflix/conductor/discussions/2805

**Comment by v1r3n** [comment]
Hi @santhosh34 if you use postgres, it uses the postgres based queues by default.  
Also dyno-queues are deprecated and newer versions use https://github.com/orkes-io/orkes-queues

**Reply by santhosh34** [reply]
Thanks @v1r3n    Currently, orkes-queues library is used as the underlying queuing infrastructure for Orkes Conductor? does it work with base variant i.e  netflix conductor as well ?

Referred:
https://orkes-conductor.slack.com/archives/C02KJ820XPW/p1681503997908319?thread_ts=1681499280.625799&cid=C02KJ820XPW

**Reply by v1r3n** [reply]
Hi @santhosh34 yes we use orkes-queues as the underlying queuing infra for Orkes Conductor.  It also works with Netflix Conductor and is used as the [default queues](https://github.com/Netflix/conductor/blob/main/server/build.gradle#L42) now.
